streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -134,77 +136,97 @@ public void testTopologyMetadata() {
     public void testDrivingSimpleTopology() {
         final int partition = 10;
         driver = new TopologyTestDriver(createSimpleTopology(partition), props);
-        driver.pipeInput(recordFactory.create(INPUT_TOPIC_1, "key1", "value1"));
-        assertNextOutputRecord(OUTPUT_TOPIC_1, "key1", "value1", partition);
-        assertNoOutputRecord(OUTPUT_TOPIC_2);
+        final TestInputTopic<String, String> inputTopic = driver.createInputTopic(INPUT_TOPIC_1, STRING_SERIALIZER, STRING_SERIALIZER, Instant.ofEpochMilli(0L), Duration.ZERO);
+        final TestOutputTopic<String, String> outputTopic1 =
+                driver.createOutputTopic(OUTPUT_TOPIC_1, Serdes.String().deserializer(), Serdes.String().deserializer());
+        final TestOutputTopic<String, String> outputTopic2 =
+                driver.createOutputTopic(OUTPUT_TOPIC_1, Serdes.String().deserializer(), Serdes.String().deserializer());
```

Comment:
This must be `OUTPUT_TOPIC_2`

Reply:
I don't understand the original test. Why it is checking OUTPUT_TOPIC_2 if it is not in Topology at all.

Reply:
I pushed the version with outputTopic2 removed

Reply:
The original intent of the test was to ensure, we don't write into non-exiting topics, ie, create a topic out of nowhere -- but with the new abstraction that cannot happen anyway I guess.